### ISSUE TITLE
Bot unable to authenticate against Jellyfin

### ISSUE THREAD

**Issue by IReclaimer** [manuel-rw/jellyfin-discord-music-bot#363]
The Music Bot is unable to authenticate against Jellyfin: ERROR [JellyfinService] Failed to authenticate with response code 200

Steps to reproduce the behaviour:
1. Start the Music Bot container.
2. Observe the logs and authentication process.

The Music Bot should be able to authenticate successfully against Jellyfin, allowing access to the media resources.

DEBUG Logs: [https://pastebin.com/cqm4Qj1g](https://pastebin.com/cqm4Qj1g)

Environment Variables:
```
DISCORD_CLIENT_TOKEN=[Discord_Token]
JELLYFIN_SERVER_ADDRESS=[http://my.internal.jellyfin.server:8096]
JELLYFIN_AUTHENTICATION_USERNAME=Jellybot
JELLYFIN_AUTHENTICATION_PASSWORD=[Complex_Password]
LOG_LEVEL=DEBUG
UPDATER_DISABLE_NOTIFICATIONS=false
```

**Comment by manuel-rw** [comment]
Are the [ ] just from removing the actual value or do you have them in your environment variables?

**Comment by IReclaimer** [comment]
Thanks for writing replying!

Everything in the square brackets '[xxx]' is just to indicate that what's shown isn't the real value. There are no square brackets in the real values.

**Comment by manuel-rw** [comment]
Do you have anything after the Jellyfin URL? Remove any ``/``, ``#`` or ``index.html`` from the URL. Example: ``https://my-jellyfin.com``

**Comment by IReclaimer** [comment]
I use the internal network FQDN of the Jellyfin server as opposed to the external domain since both are on the same network. The internal FQDN is: `http://jellyfin.lab.home.arpa:8096`

I assume the port is needed and isn't causing an issue? Unless it absolutely has to be the external network name?

**Comment by manuel-rw** [comment]
No, that shouldn't be an issue. Is there a proxy between the bot and Jellyfish?

**Comment by IReclaimer** [comment]
No, there's no proxy between the box and jellyfin. They're both on the same subnet.

There is an Nginx instance on the network which Jellyfin uses as a _reverse_ proxy, but that shouldn't come into play unless I used the external domain name to access it.

Is there anything I can do to gather more helpful information for you?

**Comment by manuel-rw** [comment]
Yes, send your Jellyfin version.
You could also try to replicate the request manually to debug further. It seems that Jellyfin is trying to return the normal UI instead of the API. The bot only works when the API is correctly handled.

**Comment by manuel-rw** [comment]
Stale, closing
